src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+
+interface ErrorNotificationProps {
+  errorMessage: string;
+  setErrorMessage: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const ErrorNotification: React.FC<ErrorNotificationProps> = ({
+  errorMessage,
+  setErrorMessage,
+}) => {
```

Comment:
```suggestion
}) => {

```

Reply:
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/167062108/4dd4ef71-a37b-460a-bd3c-c1e824bbe899)
i can't do this